cli/js/lib.deno.ns.d.ts

```diff
@@ -33,69 +33,67 @@ declare namespace Deno {
    * when `Deno.runTests` is used */
   export function test(name: string, fn: TestFunction): void;
 
-  enum TestStatus {
-    Passed = "passed",
-    Failed = "failed",
-    Skipped = "skipped"
-  }
-
-  interface TestResult {
-    name: string;
-    status: TestStatus;
-    duration?: number;
-    error?: Error;
-  }
-
-  interface TestStats {
-    filtered: number;
-    ignored: number;
-    measured: number;
-    passed: number;
-    failed: number;
-  }
-
-  export enum TestEvent {
+  enum TestEvent {
```

Comment:
Why not under namespace?

Reply:
I think it would have to be public then (subject to other discussions).